compiler/quilt/nodes.py

```diff
@@ -57,17 +55,24 @@ def _data(self):
             elif isinstance(self._node, core.FileNode):
                 self.__cached_data = store.get_file(self._node.hashes)
             else:
-                # XXX: This is wrong.
-                hash_list = list(core.find_object_hashes(self._node, sort=True))
-                self.__cached_data = store.load_dataframe(hash_list)
+                assert False
```

Comment:
is this caught and rolled up into an error message? if not we should at least have a descriptive string here

Reply:
No - but this can't happen. I.e., it's a bug in the code, not a user error, if it does happen.